index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Conference Page</title>
+    <link rel="stylesheet" href="/css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/716817a6e7.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="page-heading">
+        <i class="fa fa-facebook-f"></i>
+        <i class="fa fa-twitter"></i>
+        <span class="heading-label">English</span>
+        <span class="heading-label">My Page</span>
+        <span class="heading-label">Logout</span>
+      </div>
+      <div class="navbar">
+        <a href="#"
+          ><img
+            src="/images/logo.png"
+            alt="Sawasawa brand logo"
+            class="logo-img"
+        /></a>
+        <nav>
+          <div class="nav-links" id="navLinks">
+            <i class="fas fa-times close-icon" id="closeBtn"></i>
+            <ul class="nav-items" id="navItems">
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">Trends</a></li>
+            </ul>
+            <button class="btn">Latest Trends</button>
+          </div>
+        </nav>
+        <i class="fas fa-bars bars" id="openBtn"></i>
+      </div>
+    </header>
+
+    <!-- Main page -->
+    <section class="main-section">
+      <div class="headline-section">
+        <div class="headline-text">
+          <span class="text">Hello! Fashion World</span>
+          <h1 class="main-heading">
+            SAWASAWA <br />
+            FASHION WEEK <br />
+            2022
+          </h1>
+          <p class="headline-para">
+            A celebration that brings together different creatives from
+            different cultures to share their uniqueness in the fashion
+            industry.
+          </p>
+        </div>
+        <div class="headline-details">
+          <span class="headline-date">2022.05.20(FRI) ~ 21(SAT)</span>
+          <p>@Noir Art Gallery, Muthangari Rd</p>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] OPTIONAL
Kindly ensure the contents of this section si stands out quite larger, as seen in the template design, and should be vertically aligned to the left with logo and the leftmost side of the contents of the other sections

Your **design:**
![image](https://user-images.githubusercontent.com/12745474/169580600-b430c9bf-4ef9-4ad5-b812-e08655da9e8f.png)

**The original design:**
![image](https://user-images.githubusercontent.com/12745474/169580787-d1ec0904-b110-47ca-8363-c58c9d1c44c3.png)